scripts/update/requestrr.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+. /etc/swizzin/sources/functions/os
+if [[ -f /install/.requestrr.lock ]]; then
```

Comment:
I'd negate this and kill the script if that's not present, seeing as this should be an upgrader. I don't see the other case handled as is.